RestructuredText/syntax_test_restructuredtext.rst

```diff
@@ -134,4 +134,19 @@ Not verbatim
     Blank lines are fine
 
     </b>
-..  ^^^^ text.html.basic
\ No newline at end of file
+..  ^^^^ text.html.basic
+
+.. math::
+
+    \sum_{i=0}^N x_i^2
+..  ^^^^ text.tex.latex support.function.math.tex
+..  ^^^^^^^^^^^^^^^^^^^ text.tex.latex meta.environment.math.block.restructuredtext
+
+
+:test:
+
+   :math:`\alpha + \beta`
+.. ^^^^^^ support.function.role.builtin.restructuredtext
+..       ^ punctuation.definition.interpreted.restructuredtext
+..        ^^^^^^^^^^^^^^ text.tex.latex meta.environment.math.inline.restructuredtext
+..                      ^ punctuation.definition.interpreted.restructuredtext
```

Comment:
With the linked patch to LaTexTools, and a config change there to also render inline math, this displays as:
![image](https://user-images.githubusercontent.com/425260/67533976-e7d40c00-f6c3-11e9-9cf5-0edddcc054db.png)